libs/domains/organizations/feature/src/lib/hooks/use-organizations/use-organizations.tsx

```diff
@@ -1,7 +1,7 @@
 import { useQuery } from '@tanstack/react-query'
 import { queries } from '@qovery/state/util-queries'
 
-export function useOrganizations() {
+export function useOrganizations({ enabled }: { enabled?: boolean }) {
```

Comment:
```suggestion
export function useOrganizations({ enabled }: { enabled?: boolean } = {}) {
```
It avoids specifying empty object every time

Reply:
https://github.com/Qovery/console/pull/977/commits/46b7b4ccdc2207ada2da4ef731d75d7e8569caae